packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.scss

```diff
@@ -4,11 +4,17 @@
   .collectionSelector {
     height: 15%;
     min-height: 2rem;
+    justify-content: space-between;
 
     span {
       color: var(--phoenix-text-color-secondary);
       margin-right: 1rem;
     }
+
+    .checkbox-wrapper {
```

Comment:
The layout you have can be achieved with flex. No need to use any other properties.